CONTRIBUTING.md

```diff
@@ -33,47 +34,62 @@
     brew install openssl
     ```
    
-4. **Generate OpenAPI client** (Optional, usually done automatically at build time)
-   
-   Pinecone uses an OpenAPI spec for control-plane operations like `create_index()`. The OpenAPI client is automatically generated using [openapi-generator](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/rust-server.md) during project build.
-   This process uses Docker to `docker run` OpenAPI's generator image.  
-   **If you don't have docker installed, or you don't want to use docker** -  you can download the generated code from the [latest release](https://github.com/pinecone-io/pinecone-client/releases). 
-   Simply extract the `index_service.zip` file into the `index_service/` folder at the root of the project.
```

Comment:
This step was not working for me previously and I had to download the generated core manually. Seems like this new setup works quite well though, nice! 
![Screenshot 2023-10-27 at 12 38 25 PM](https://github.com/pinecone-io/pinecone-client/assets/119623786/a4a038c3-75e8-45bc-92aa-bf0fb0a9b432)

![Screenshot 2023-10-27 at 12 53 38 PM](https://github.com/pinecone-io/pinecone-client/assets/119623786/ff4b5fd1-54c0-48bb-a921-bd9f1fa59977)